### ISSUE TITLE
WIP: Media and Entertainment Interest Group rechartering

### ISSUE THREAD

**Issue by ashimura** [w3c/strategy#378]
New charter proposal, reviewers please take note.

# Charter Review

[Draft Charter:](https://w3c.github.io/media-and-entertainment/charters/charter-2023.html)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing IG recharter
* [diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2021%2F06%2Fme-ig-charter.html&doc2=https%3A%2F%2Fw3c.github.io%2Fmedia-and-entertainment%2Fcharters%2Fcharter-2023.html)

The draft charter above is still under discussion by the IG.
* * [GitHub repo for the discussion](https://github.com/w3c/media-and-entertainment)

---
Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, and security. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).**

Communities suggested for outreach: 

Known or potential areas of concern: 

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)

Anything else we should think about as we review?

**Comment by plehegar** [comment]
Advance notice: https://lists.w3.org/Archives/Member/w3c-ac-members/2023AprJun/0011.html

**Comment by ruoxiran** [comment]
https://github.com/w3c/apa/issues/324

**Comment by himorin** [comment]
editorial: latest two entry (except one line for this) in history table has wrong link?

**Comment by himorin** [comment]
No comment nor request from i18n

**Comment by matatk** [comment]
From APA: We approve the MEIG charter. We filed [a separate issue on the MEIG repo](https://github.com/w3c/media-and-entertainment/issues/97) which also proposes a liaison between our groups. For convenience, here's a quote of that issue, filed by @JaninaSajka...

> The Accessible Platform Architectures (APA) Working Group, together with our Research Questions Task Force (RQTF) have reviewed this proposed Charter and are pleased to support it. Only if you agree, we would prefer a formal liaison and would like to name APA Co-Chair Janina Sajka as our liaison. We are also agreed among ourselves that it is time for us to update the [Media Accessibility User Requirements (MAUR)](http://www.w3.org/TR/media-accessibility-reqs/). To that end it may be helpful to have participation from one or more participants from the IG helping with this update, as we expect a major revision to encompass multiple opportunities we believe to be available now that were not available when MAUR was published in 2015. Also, if you are in agreement, we would appreciate having this update listed in the Scope section, i.e. "such as an update to the Media Accessibility User Requirements 1.0." In other words, while we will take responsibility for MAUR and also the lead in updating this document, we would prefer to develop the new version in close collaboration with the IG. We do note that MAUR 2.0 may also suggest certain enhancements to existing W3C specifications, or to HTML which may be out of scope for the IG, and that APA should perhaps publish the Note itself, but with the IG's technical support to the extent commensurate with your Charter.

**Comment by svgeesus** [comment]
No comments except _Thank you for doing a good job on this charter_

+1 to send to the AC once horizontal review is complete

**Comment by chrisn** [comment]
Thank you @svgeesus. We have updated the charter to describe the expected collaboration with APA WG, as suggested by @JaninaSajka and @matatk.

**Comment by plehegar** [comment]
AC Review: https://lists.w3.org/Archives/Member/w3c-ac-members/2023JulSep/0029.html

**Comment by plehegar** [comment]
Some links need to be updated. See https://github.com/w3c/media-and-entertainment/pull/104

**Comment by plehegar** [comment]
#chrisn , please use the [W3C calendar](https://www.w3.org/groups/ig/me/calendar/) to track IG meetings. At the moment, it's a guess game for outsiders to see where the IG meetings are announced, where past meetings happened, etc. If you need a walk around on how to use the calendar system, I'm happy to give you one.

**Comment by himorin** [comment]
editorial: 

- opening `<dl>` seems missing at right after `<h3 id="w3c-coordination">W3C Groups</h3>`
- `performance` HR shall be removed from coordination section

**Comment by chrisn** [comment]
@plehegar On the calendar, we do use it, but don't have a recurring entry for future meetings. That could be useful. On the homepage, I'll discuss with @ashimura and co-chairs and let you know.

@himorin Thank you, we'll update the charter to fix those editorial issues.

**Comment by plehegar** [comment]
Announced: https://lists.w3.org/Archives/Member/w3c-ac-members/2023OctDec/0023.html
